File: JavaProject/Payments.java

public class Payments {
	private Payment[] all_payments;
	private double actual_card_payments;
	private double actual_cash_payments;
	
	public void PayOut_salary(String employee_ID){
		
	}
	
	public void Take_payment(double value, boolean card){
	}
	
	public void Give_bonus(String employee_id, double bonus){
		
	}
	
	public void Sum_actual_payment_for_daily_report(){
		
	}
	
	public Payment[] getAll_payments() {
		return all_payments;
	}
	
	public void setAll_payments(Payment[] all_payments) {
		this.all_payments = all_payments;
	}
	
	public double getActual_card_payments() {
		return actual_card_payments;
	}
	
	public void setActual_card_payments(double actual_card_payments) {
		this.actual_card_payments = actual_card_payments;
	}
	
	public double getActual_cash_payments() {
		return actual_cash_payments;
	}
	
	public void setActual_cash_payments(double actual_cash_payments) {
		this.actual_cash_payments = actual_cash_payments;
	}

}

File: CSharpProject/Hotel/Model/Payments.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hotel.Model
{
    class Payments
    {
        List<Payment> allPayments;
        double actualCash;
        double actualCard;

        public Payments()
        {
            allPayments = new List<Payment>();
            actualCard = 0.0;
            actualCash = 0.0;
        }

        public void PayOutSalary(String employeeID, double salary)
        {
            throw new NotImplementedException();
        }

        public void TakePayment(double paymentValue, bool card)
        {
            throw new NotImplementedException();
        }

        public void GiveBonus(String employeeID, int percentageBonus)
        {
            throw new NotImplementedException();
        }

        //------------------ GETTERS AND SETTERS-----------------

        public Payment this[int i]
        {
            get { return allPayments[i]; }
            set { allPayments[i] = value; }
        }

        public double ActualCash
        {
            get { return actualCash; }
            set { actualCash = value; }
        }

        public double ActualCard
        {
            get { return actualCard; }
            set { actualCard = value; }
        }
    }
}
